packages/react-charting/src/types/ISankeyDataPoint.ts

```diff
@@ -0,0 +1,80 @@
+import { SankeyLink, SankeyNode } from 'd3-sankey';
```

Comment:
revert this change. This is a breaking change

Reply:
done